app/controllers/cmsimple/pages_controller.rb

```diff
@@ -26,7 +26,7 @@ def update_content
     end
 
     def editor
-      render :nothing => true, :layout => 'editor'
+      render :text => '', :layout => 'editor'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.